src/api/web/endpoint/package.js

```diff
@@ -45,6 +45,7 @@ function addPackageWebApi(route: Router, storage: IStorageHandler, auth: IAuth)
               permissions.push(pkg);
             }
           } catch (err) {
+            console.error(err)
```

Comment:
Problem is the eslint don't allow `console.error` on code. Here we should use the logger object.

```
/home/circleci/verdaccio/src/api/web/endpoint/package.js
  48:13  error  Unexpected console statement  no-console
  48:31  error  Missing semicolon             semi
```

Instead I would

```
import logger from '../../../lib/logger';

// below 
logger.logger.error({name: pkg.name, error }, 'permission process for @{name} has failed: @{error}');
```

Reply:
Ok, will change it in about 1 - 2 hours.

Reply:
Do we run the eslint checks or how can I make sure to catch such things like you mention?

Reply:
If you remove the `console.error(err)` all should be fine.  But you can run locally `yarn lint` to check all lint pass.  https://github.com/verdaccio/verdaccio/wiki/Build-Source-Code#scripts-1

Reply:
Ag'h, CI is red because of this which is good =). I guess I either oversaw it or it took some time after my changes.